docs/reference/rest-api.md

```diff
@@ -7,14 +7,6 @@ position: 9
 
 All the API endpoints start with the `api/v1/` prefix. `v1` indicates that we are currently using version 1 of the API.
 
-
-## Format
```

Comment:
indeed it was obsolete. It's still the case for the section `Error handling`. Can you update it? this  error handling is only for non stream request.